io.github.getnf.embellish.yml

```diff
@@ -0,0 +1,37 @@
+id: io.github.getnf.embellish
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: embellish
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --filesystem=xdg-data/embellish:create
+  - --filesystem=xdg-data/fonts:create
```

Comment:
Why do you need this? Fonts from here are automounted by Flatpak.

Reply:
For the same reason of the GUI being compatable with the TUI/CLI build. 

Also, will fonts automounted by flatpak be acceessible by non-flatpak apps?

Reply:
> Also, will fonts automounted by flatpak be acceessible by non-flatpak apps?

Not sure what you mean, the sandbox is only for this app, the font files are already on host, flatpak will bind mount them inside the sandbox.

As the fonts are already on host they are accessible by other host apps.

Reply:
Oh i misunderstood you.

I need access to the fonta directory because my tool is used to install and uninstall fonta from the system.

My tool won't be able to function without this access.

Reply:
Does `--filesystem=~/.local/share/fonts` work or do you need it to be xdg-data/fonts?

Reply:
`--filesystem=~/.local/share/fonts` does work

Reply:
So can you change this one and drop the other one?